userspace/falco/configuration.cpp

```diff
@@ -380,11 +443,14 @@ void falco_configuration::load_yaml(const std::string& config_name, const yaml_h
 	/* We put this value in the configuration file because in this way we can change the dimension at every reload.
 	 * The default value is `4` -> 8 MB.
 	 */
-	m_syscall_buf_size_preset = config.get_scalar<uint16_t>("syscall_buf_size_preset", 4);
-
-	m_cpus_for_each_syscall_buffer = config.get_scalar<uint16_t>("modern_bpf.cpus_for_each_syscall_buffer", 2);
-
-	m_syscall_drop_failed_exit = config.get_scalar<bool>("syscall_drop_failed_exit", false);
+	// TODO: remove in Falco 0.38 since they are deprecated.
+	m_kmod.m_buf_size_preset = config.get_scalar<uint16_t>("syscall_buf_size_preset", 4);
```

Comment:
We load `m_buf_size_preset` and `m_drop_failed_exit` 3 times, one for each supporting driver; this is just to avoid a switch on `m_driver_mode`.
It will be dropped in Falco 0.38 anyway.